configuration/Makefile

```diff
@@ -11,6 +11,12 @@ RNDR ?= ../../rndr/.bin/rndr # DEBUG
 
 JSONNET_SRC = $(shell find . -type f -not -path './*vendor/*' \( -name '*.libsonnet' -o -name '*.jsonnet' \))
 
+UNAME := $(shell uname)
+ifeq ($(UNAME), Darwin)
+   # Overcome Mac default limiting xargs -I replace size.
+   XARGS_FLAGS ?= -S 4096
```

Comment:
Alternatively, we could just look for GNU version of `xargs` on macOS and use that using something like `XARGS ?= $(shell which gxargs 2>/dev/null || which xargs)` (you can install `gxargs` with `brew install findutils`).

But this works as well, so I don't feel too strongly either way! 🙂

Reply:
I agree with saswata, it's probably better to prompt users to use gnu `findutils`.

Reply:
@moadz The Makefile's behavior on unmodified Mac/BSD is bizarre and it will be hard to communicate the solution (`brew install findutils`) using @saswatamcode 's idea to have `gxargs` override `xargs` -- but only if present.

This is similar but will fail in an obvious way:

```bash
ifeq ($(UNAME), Darwin)
   # macOS/BSD xargs not compatible with this Makefile (because of -S replsize default of 256)
   XARGS ?= gxargs
else
   XARGS ?= xargs
endif
```

Alternately we could use the work-around in this PR which does not require installing anything.

Reply:
If one is installing `findutils` they're most likely adding it to their `PATH` as well. 

I like that your approach requires little intervention on behalf of the user. But, I also think in this case where GNU/BSD `xargs` (and other tools like `sed`)  have diverged significantly, it is more useful that a contributor is aware of explicit GNU compatibility requirement on BSD. This means any new changes + steps can be done without having to consider BSD backwards compatibility.

Reply:
We have talked about three approaches:
- My first approach, which detects Macs and adds BSD-specific flags
- My second approach https://github.com/observatorium/observatorium/pull/471#discussion_r845008628 which requires Mac users to install GNU _xargs_ as _gxargs_
- @saswatamcode 's approach which uses gxargs if present.

I believe my second approach best addresses @moadz 's comments.  I will revise the PR toward that approach.